generator/src/main/java/com/scottlogic/deg/generator/walker/reductive/fieldselectionstrategy/FieldValue.java

```diff
@@ -0,0 +1,27 @@
+package com.scottlogic.deg.generator.walker.reductive.fieldselectionstrategy;
+
+import com.scottlogic.deg.generator.Field;
+
+public class FieldValue {
```

Comment:
Hmm it feels like there's a neater way of doing this?

Reply:
no tuples in java.

otherwise, possibly, but i've not thought of it yet